CMSgov/cmcs-eregulations
# EREGCSC-1607 -- Feedback banner

Resolves [EREGCSC-1607](https://jiraent.cms.gov/browse/EREGCSC-1607)

**Description**

Changes Flash Banner at top of page from linking to About page to instead prompting user to give feedback.

Feedback is collected via a Google Forms survey that is displayed to the user in an iframe in a blocking modal that is overlaid on top of the app.

**This pull request changes**

- Replaces link to `/about` in the Flash Banner at top of page with a link to a feedback survey
- creates `survey_url` parameter in AWS Parameter Store to be used for url to feedback survey (instead of hard coding the URL in the codebase)
- Creates `BlockingModal` and `IFrameContainer` Vue components that can be used in Django templates or in the Vite SPA pages (`/search`, `/resources`).
- Ensures BlockingModal is accessible and traps scrolling and focus in the modal content
- Adds basic Happy Path end to end test for Blocking Modal
- Fixes a few cypress-related issues

**Steps to manually verify this change**

1. Ensure that there is a `/eregulations/survey_url` parameter in AWS Parameter Store for `dev`, `val`, and `prod` environments
2. Visit [experimental deployment](https://8ma5bno5c0.execute-api.us-east-1.amazonaws.com/dev653) and click the "give us feedback" link at the top of the homepage and make sure things match the [Figma comp](https://www.figma.com/file/y6fnov0MZS0ognkkWogZbA/eRegs-Global-Feedback?node-id=1736%3A511&t=LMOIr1a3YCXEfTKs-0)
3. Visit all parts of the app (ToC, Part view, Search, Resources, About) and ensure the Flash Banner appears on all pages and the survey appears in a blocking modal
